easybuild/easyconfigs/r/Raven/Raven-1.8.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,53 @@
+easyblock = "CMakeNinja"
+
+name = "Raven"
+version = "1.8.1"
+
+homepage = "https://github.com/lbcb-sci/raven"
+description = "Raven is a de novo genome assembler for long uncorrected reads."
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+source_urls = ["https://github.com/lbcb-sci/raven/archive/refs/tags/"]
+sources = ["%(version)s.tar.gz"]
+checksums = ['00009e9027761fa28313db4e2b5c5f40ebc10e6631b43b755e321ac6bbcc2c06']
+
+builddependencies = [
+    ('CMake', '3.22.1'),
+    ('Ninja', '1.10.2'),
+]
+
+preinstallopts = "export CMAKE_BUILD_PARALLEL_LEVEL=%(parallel)s && "
+configopts = ['-DRAVEN_BUILD_EXE=1 -G Ninja ']
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('zlib', '1.2.11'),
+]
+
+exts_defaultclass = 'PythonPackage'
+
+exts_default_options = {
+    'download_dep_fail': True,
+    'use_pip': True,
+}
+
+exts_list = [
+    (name, version, {
+        'modulename': 'ravenpy',
+        'source_urls': ['https://github.com/lbcb-sci/raven/archive/refs/tags/'],
+        'sources': ['%(version)s.tar.gz'],
+        'checksums': ['00009e9027761fa28313db4e2b5c5f40ebc10e6631b43b755e321ac6bbcc2c06'],
+    }),
+]
+
+modextrapaths = {'PYTHONPATH': 'lib/python%(pyshortver)s/site-packages'}
+
+sanity_check_paths = {
+    'files': ['bin/raven'],
+    'dirs': []
+}
+
+sanity_check_commands = ["pip check"]
```

Comment:
Could you also add a simple `sanity_check_command` for the `raven` binary itself, e.g. `raven --help` or `raven --version`?